.gitmodules

```diff
@@ -1,17 +1,18 @@
 [submodule "wxWidgets"]
 	path = wxWidgets
-	url = https://github.com/RPCS3/wxWidgets.git
+	url = https://github.com/Zangetsu38/wxWidgets
 	ignore = dirty
```

Comment:
Alright I send a pull-request upstream https://github.com/wxWidgets/wxWidgets/pull/48 once that is merged I will rebase our little silly "add setup.h" change on top of that in RPCS3/wxWidgets